src/relay/pass/fold_scale_axis.cc

```diff
@@ -391,16 +387,21 @@ Expr MultiplyForwardRewrite(const Call& ref_call,
   Expr lhs = new_args[0];
   Expr rhs = new_args[1];
   auto rnode = make_node<ScaledExprNode>();
-  if (MatchBroadcastToLeftAxes(tlhs, trhs, expected_out_axes, &rhs)) {
+  if (MatchBroadcastToLeftAxes(tlhs, trhs, expected_out_axes, &rhs) &&
```

Comment:
Though the positiveness / constantness check here is unnecessary if relu is not present, I think it is okay given the actual workload (typically bn+relu) we care about